app/lib/actions/katello/product/repositories_certs_reset.rb

```diff
@@ -0,0 +1,25 @@
+module Actions
+  module Katello
+    module Product
+      class RepositoriesCertsReset < Actions::AbstractAsyncTask
+        def plan(product)
+          ssl_ca_cert_id = product.ssl_ca_cert_id
+          ssl_client_cert_id = product.ssl_client_cert_id
+          ssl_client_key_id = product.ssl_client_key_id
+          # Plan Repository::Update only for repositories which have different certs key
+          product.repositories.each do |repo|
```

Comment:
it looks like the UI only allows yum repos to set ssl values.  Is that intentional?  If so we probably need to adjust it here (and elsewhere) to restrict to yum repos.  If not, we should probably update the UI to allow use on other repo types.

Reply:
Otherwise this works well!

Reply:
You are right. Only yum repos supports ssl values but katello database doesn't differentiate. The same logic is used for Gpg Key which I have just re-used and adapted. I tested with yum, deb, file and puppet repo type and everything works as expected.

Reply:
ahh, okay, how about we update this here to: product.repostories.yum_type.each for now and open an issue for the other issues? 

And you're positive that pulp doesn't support the ssl options for non-yum types?

Reply:
You are right, again. The Feed SSL Certs are available for all pulp types. I would suggest, that I will add a new issue to include all feed ssl cert fields to the other repo types. My fault (some month ago). Thanks you very much to raise this.

Reply:
Ahh great! okay, so lets merge this as it is now.  And we can fix the creation/details pages.

Reply:
Rebased.

Follow up task to support other pulp repo types: https://projects.theforeman.org/issues/24111